Lidong-Huang/Censored.DF
# Main_Dist_fitting_Weighting_Substitution

# Description

This pull request includes the following updates and improvements to the Distribution Fitting for Left Censored Data application:

## Features Added
- Added a new module for Weighting Substitution Method.
- Users can now select distribution type (Lognormal or Gamma) and calculate arithmetic and geometric means based on LOD/2 substitution method.
- Updated CSS for better visual layout, ensuring that the title wraps correctly and the footer is positioned properly.

## Bug Fixes
- Fixed issues with the .Rhistory file being included in commits.
- Ensured that the app handles left-censored data correctly and displays Shapiro-Wilk statistics and P-value from the `EnvStats` package.

## Improvements
- Enhanced user instructions to include details about the new Weighting Substitution module.
- Improved the readability and structure of the instructions.
- Optimized the data fitting process and ensured accurate calculations for both Lognormal and Gamma distributions.

# How to Test
1. Upload a CSV file with two columns: "Data" and "Censored".
2. Select the distribution type from the Weighting Substitution module.
3. Click "Calculate" to see the arithmetic and geometric means, as well as the standard deviation.
4. Check the Summary of Fitting and CDF & Q-Q Plot tabs to verify the results.

# Related Issues
- Resolves issue #1: Addition of Weighting Substitution module.
- Resolves issue #2: CSS improvements for title and footer layout.

Please review the changes and provide feedback or approval for merging.
